src/Hydrate/EntityHydrator.php

```diff
@@ -121,7 +121,7 @@ private function hydrateEntity(string $entityName, array $entityRaw, array $data
         $relationships = $entityRaw['relationships'] ?? [];
 
         // reserve cache before relationships hydration. This prevents circular references to fail
-        if($this->useCache && isset($cacheKey)) {
+        if($this->useCache) {
```

Comment:
`$cacheKey` will always be set, because it is set within an existing `if($this->useCache)` block.
PHPStan was complaining about that: https://github.com/TobiasGraml11/shopware-php-sdk/actions/runs/3730960214/jobs/6328616982